### DISCUSSION TITLE
Find a json file produced by excecutind an exe in Github worlflow

### DISCUSSION THREAD

**Discussion by alaabet** [community/community#54872] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

In my Github workflow an exe of a WPF is excecuted which produces a json file. My issue is to locate json file to be parsed.

**Comment by hrishikesh-kocharekar-dev** [comment]
If you're running a GitHub workflow that generates a JSON file using an executable of a WPF application, you'll need to make sure that the JSON file is saved to a location where you can access it from within your workflow.

By default, the current working directory of a GitHub Actions workflow is the root of the repository. So if you're generating the JSON file in the WPF application and saving it to the current working directory, you should be able to access it from within your workflow using a relative path.

For example, let's say your WPF application generates a file named `output.json` and saves it to the root of the repository. You could then use a command like `cat output.json` or `echo "Hello $(cat output.json)"` in your workflow to access the contents of the JSON file.

If your JSON file is being saved to a different location, you'll need to modify your workflow to navigate to that location before attempting to read or parse the file. You can use standard shell commands like `cd` to change the working directory or `ls` to list the contents of a directory.

For example, if your WPF application is saving the JSON file to a directory named `data` at the root of the repository, you could use the following commands in your workflow to navigate to the directory and parse the JSON file:
cd data
cat output.json | jq '.'

This assumes that you have the `jq` command-line tool installed in your workflow environment, which is a useful tool for working with JSON data.

I hope this helps!

**Reply by alaabet** [reply]
Thank you @hrishikesh-kocharekar-dev, but I couldn't see that the json file is saved. I use: run: ./project-name/bin/debug/xxx.exe
xxx.exe should produce a json file in the same path, but I couldn't find it.
Do I need to set any permisson to allow saving the json file?
